_sub/network/aws-lb-controller/main.tf

```diff
@@ -0,0 +1,69 @@
+# --------------------------------------------------
+# Create JSON files to be picked up by Flux CD
+# --------------------------------------------------
+resource "github_repository_file" "helm" {
+  repository = var.repo_name
+  branch     = local.repo_branch
+  file       = "${local.cluster_repo_path}/${local.app_install_name}-helm.yaml"
+  content = templatefile("${path.module}/values/app-helm.yaml", {
+    app_install_name = local.app_install_name
+    deploy_name      = var.deploy_name
+    namespace        = var.namespace
+    helm_repo_path   = local.helm_repo_path
+    prune            = var.prune
+  })
+  overwrite_on_create = true
+}
+
+resource "github_repository_file" "helm_install" {
+  repository = var.repo_name
+  branch     = local.repo_branch
+  file       = "${local.helm_repo_path}/kustomization.yaml"
+  content = templatefile("${path.module}/values/helm-install.yaml", {
+    gitops_apps_repo_url    = var.gitops_apps_repo_url
+    deploy_name             = var.deploy_name
+    gitops_apps_repo_branch = var.gitops_apps_repo_branch
+  })
+  overwrite_on_create = true
+}
+
+resource "terraform_data" "aws-load-balancer-mutating-webhook-cleanup" {
+  input = var.kubeconfig_path
+
+  provisioner "local-exec" {
+    environment = {
+      K_CONFIG = self.input
+    }
+    when    = destroy
+    command = "kubectl --kubeconfig=$K_CONFIG delete MutatingWebhookConfiguration aws-load-balancer-webhook --ignore-not-found=true"
+  }
+  depends_on = [ github_repository_file.helm_install ]
+}
+
+resource "terraform_data" "aws-load-balancer-validating-webhook-cleanup" {
```

Comment:
This is only needed if we remove it and this is only in to satisfy the QA test pipeline? I vote to remove it to simplify things.